config/data_scrubber.go

```diff
@@ -17,11 +19,19 @@ var (
 		"secret", "credentials", "stripetoken"}
 )
 
+const (
+	defaultCacheTTL = 25
```

Comment:
nit: this name is confusing to me because it's not really a "time" but the number of scrubs in a cache period. can we rename or add a comment (or both, if necessary)